src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -57,7 +57,7 @@ const DataProcessingPage = ({ experimentId, experimentData, route }) => {
     {
       key: 'dataIntegration',
       name: 'Data integration',
-      render: (key) => <DataIntegration filtering key={key} />,
```

Comment:
what is this `filtering` used for? I saw it in a couple of other places, but I didn't see where it is getting set from

Reply:
the `filtering` key is not used, to it should be removed

Reply:
can I remove it in the other filters too?

Reply:
no point having things that are not used

Reply:
From what I see, they were used as a flag to enable / disable one of the plot styling configs in the old data processing components. I think you can leave them be at the moment. They should be removed later on as the components are being worked on,